### ISSUE TITLE
exp/clients/horizon: _links templates do not contain `include_failed`

### ISSUE THREAD

**Issue by vcarl** [stellar/go#1006]
This option is supported by `/transactions` and several other endpoints, but doesn't appear in the URL template. For example, I'd expect transactions for account to include it.

https://horizon-testnet.stellar.org/accounts/GCDLRUXOD6KA53G5ILL435TZAISNLPS4EKIHSOVY3MVD3DVJ333NO4DT

**Comment by bartekn** [comment]
Yes, let's discuss this first. I'm worried that if this becomes a param in URL template and there's a bug in a parser, this will become `true` somehow. This sound paranoid but such situation will have a very bad outcome. So I'm wondering if it's better to leave this parameter hidden by default.
